srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml

```diff
@@ -938,13 +939,64 @@ SRTP packets                 <------->      SRTP packets
          <section anchor="iana-ekt-msg-types" title="EKT Message
                                                      Types">
            
-           <t> TODO IANA registry for Message Type. </t>
+           <t> IANA is requested to create a new registry for "EKT
+           Messages Types". The initial values in this registry are:
+           </t>
 
+           <texttable anchor="EKTMsgTypeTable" title="EKT
+                                                      Messages Types">
+             <ttcol align='left'>Message Type</ttcol>
+             <ttcol align='right'>Value</ttcol>
+             <ttcol align='left'>Specification</ttcol>
+             <c> Short </c><c> 0 </c><c> RFCAAAA </c> 
+             <c> Full </c><c> 2 </c><c> RFCAAAA </c> 
+             <c> Reserved </c><c> 63 </c><c> RFCAAAA </c> 
+             <c> Reserved </c><c> 255 </c><c> RFCAAAA </c> 
+           </texttable>
+           
+           <t> Note to RFC Editor: Please replace RFCAAAA with the RFC
+           number for this specification. </t>
+
+           <t> New entries to this table can be added via
+           "Specification Required" as defined in <xref
+           target="RFC5226"/>. When requesting a new value, the
+           requestor needs to indicate if it is mandatory to
+           understand or not. If it is mandatory to understand, IANA
+           needs to allocate a value less than 64, if it is not
+           mandatory to understand, a value greater than or equal to
+           64.  IANA SHOULD prefer allocation of even values over odd
```

Comment:
' if it is not mandatory to understand, a value greater than or equal to 60' needs to be allocated. 

Sentence is incomplete